siliconcompiler/report/streamlit_viewer.py

```diff
@@ -211,15 +218,18 @@ def show_file_preview(header_col_width=0.89):
         streamlit.image(path)
     else:
         try:
-            with open(path, 'r') as file:
-                content = file.read()
+            if compressed_file_extension == '.gz':
+                file = gzip.open(path, 'rt')
+            else:
+                file = open(path, 'r')
+            content = file.read()
+            file.close()
```

Comment:
`file` is an predefined object in Python, could you rename it to something else? maybe `fid` (not great but avoids the naming collision)?